conjur/api/api.py

```diff
@@ -221,6 +225,24 @@ def get_variables(self, *variable_ids) -> dict:
 
         return remapped_keys_dict
 
+    def create_token(self, create_token_data: str) -> requests.Response:
+        """
+        This method is used to create token/s for hosts with restrictions.
+        """
+        if create_token_data is None:
+            raise MissingRequiredParameterException('create_token_data cannot be empty!')
```

Comment:
@eranha is this a user facing log? If so, this doesn't really explain anything

Reply:
@sigalsax I consider it as an argument exception (runtime so to speak) it is intended for the developer